content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,211 @@
+:guilabel:`Encoding`, and :guilabel:`Barcode Pattern` for the *rules* and *barcode patterns* that
+Odoo supports for the default nomenclature.
+
+The :ref:`table <barcode/operations/default-nomenclature-list>` contains all the information that
+can be condensed with a |UPC|/|EAN| barcode.
+
+.. _barcode/operations/use-upc-ean-barcodes:
+
+Use UPC/EAN barcodes in Odoo
+============================
+
+For product identification using |UPC|/|EAN| barcodes in Odoo, businesses **must** obtain `barcodes
+<https://www.gs1us.org/upcs-barcodes-prefixes/how-to-get-a-upc-barcode>`_ purchased directly from
+GS1.
+
+The |UPC| and |EAN| barcode formats are predominantly used in their own regions. |UPC| is used only
+in the United States and Canada, while |EAN| is used everywhere else globally.
+
+A |UPC| is usually a 12-digit barcode used to identify most products, while |EAN| barcodes use
+13-digit barcodes to identify products.
+
+|UPC| codes can be converted to |EAN| by prefixing them with a zero. In Odoo, by default,
+|UPC|/|EAN| barcodes are automatically converted one way or another when trying to match a rule with
+the other encoding.
+
+Refer to the :ref:`default nomenclature list <barcode/operations/default-nomenclature-list>` to see
+a comprehensive list of all barcode patterns and rules to follow.
+
+Create rules
+------------
+
+|UPC| and |EAN| barcodes contain specific information in the barcode. Scanning these codes from the
+:ref:`default nomenclature list <barcode/operations/default-nomenclature-list>` automatically fills
+corresponding data in the Odoo database.
+
+Adding new barcode rules to this list ensures that non-standard (:dfn:`user-created`) formats are
+interpreted accurately.
+
+To create new rules, first activate :ref:`developer mode <developer-mode>`. Then, navigate to
+:menuselection:`Inventory app --> Configuration --> Barcode Nomenclatures`. Select the
+:guilabel:`Default Nomenclature` list item.
+
+On the :guilabel:`Default Nomenclature` page, select :guilabel:`Add a line` at the bottom of the
+table, which opens a :guilabel:`Create Rules` pop-up window to create a new rule.
+
+.. image:: barcode_nomenclature/barcode-nomenclature-new-rule-popup.png
+   :align: center
+   :alt: Create Rules pop-up window on Default Nomenclature page.
+
+The :guilabel:`Rule Name` field is used internally to identify what the barcode represents.
+
+The :guilabel:`Sequence` represents the priority of the rule; meaning the smaller the value, the
+higher the rule appears on the table.
+
+The barcode :guilabel:`Type` represents different classifications of information that can be
+understood by the system (e.g., :guilabel:`Package`, :guilabel:`Lot`, :guilabel:`Location`,
+:guilabel:`Coupon`, etc.).
+
+The :guilabel:`Encoding` specifies which encoding the barcode uses; this rule will only apply if the
+barcode uses this specific encoding. The available :guilabel:`Encoding` options are **EAN-13**,
```

Comment:
```suggestion
barcode uses this specific encoding. The available :guilabel:`Encoding` options are: **EAN-13**,
```
if these options appear on the UI (if chosen), wouldn't a GUI label be used in this instance? What are your thoughts?

Reply:
I went back and forth between the two, but yeah I agree with you - so I've made the change. Suggestion implemented ✅